src/components/Toaster/__tests__/ToasterProvider.test.tsx

```diff
@@ -250,3 +250,54 @@ describe('api.removeAll', () => {
         expect(screen.queryByRole('heading', {name: 'Test Toast2'})).not.toBeInTheDocument();
     });
 });
+
+describe('api.has', () => {
+    it('should return false when toast is not added', () => {
+        const providerAPI = setup();
+        expect(providerAPI.has('unexisted toasts')).toBeFalsy();
```

Comment:
toBe(false)

you interested in returning boolean from api, not some falsy value

Reply:
Fixed!